conf/messages

```diff
@@ -148,9 +148,9 @@ gmp.year = Year
 gmp.inflation_proofed = Enter the date.
 gmp.caption = Recent calculations
 gmp.revaluation.rate = Revaluation rate
-gmp.revaluation_rate.rate_held_by_hmrc = The rate held by HMRC
+gmp.revaluation_rate.rate_held_by_hmrc = HMRC's rate
```

Comment:
Are we not scrapping contractions? Should this be `HMRC is`

Reply:
"In this context this isn't technically a contraction" Says content designer